wavefront/values.yaml

```diff
@@ -23,6 +23,13 @@ collector:
     tag: 1.2.6
     pullPolicy: IfNotPresent
 
+  ## Set the updateStrategy for the collector deployment or daemonset
```

Comment:
The readme says the default is nil, but we have this uncommented here. Could this be commented out and those who need it can set it as needed?

Reply:
Done